### ISSUE TITLE
[Improvement] Support for navigating to implementation 

### ISSUE THREAD

**Issue by gayanper** [angelozerr/jdt-codemining#70]
Today we can navigate to reference by clicking on it. But that is not available for implementations. Its will be some what complete if we can open the implementation popup when we click the implementations.

**Comment by angelozerr** [comment]
If I understand your idea, click on implementation should open a popup like we have with Ctl+T. Is that?

**Comment by gayanper** [comment]
Yes exactly, but my doubt is we already have this "Open Implementation" when we CTRL+Click on the interface name as well.

**Comment by gayanper** [comment]
@angelozerr 
I started implementing this and while implementing found out that the count is shown based on the hierarchy of subtypes. But in eclipse searching to implementors/implementations give only the classes directly implementing the given interface. So don't you think its best to implement same as in eclipse search ?

ex:
Fooable <- Foo <|-- Bar

in Codemining it consider both Foo and Bar as implementations of Fooable
In Eclipse only Foo is considered.

**Comment by gayanper** [comment]
@mickaelistria @vogella any update on this ? What do you guys think ?

**Comment by angelozerr** [comment]
@gayanper I suggest you that you see https://bugs.eclipse.org/bugs/show_bug.cgi?id=541390 

Now  JDT hosts Java references, implementations code minings, I think your issue become important. It should very great if you could contribute to JDT with your work in https://bugs.eclipse.org/bugs/show_bug.cgi?id=541390

Thanks!
